src/view.tree.js

```diff
@@ -93,9 +93,25 @@ class TreeView {
     return str && str.replace(/[<>'"&]/g, '-');
   }
 
+  cmpVersions(a, b) {
```

Comment:
Why is this named "compareVersion"? This is supposed to compare files. Version is just a part of it.